pkg/ocibundle/tools/loop.go

```diff
@@ -9,14 +9,32 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
 	"github.com/sylabs/singularity/pkg/util/loop"
+	"github.com/sylabs/singularity/pkg/util/singularityconf"
 )
 
+func getMaxLoopDevices() int {
```

Comment:
This `getMaxLoopDevices` is repeated in many locations - please instead define it once as an exported/Public function (probably in `pkg/util/loop` is a good place - and then call it from other places it is needed.